MSYS2/gtk3dll.inc

```diff
@@ -1,3 +1,4 @@
+libcairo-gobject-2.dll
 libepoxy-0.dll
 libgdk-3-0.dll
 libgtk-3-0.dll
```

Comment:
Maybe add a newline at the end of this file.
It was missing already, not changed by this PR.

Reply:
Understand that is my preference too (all txt file have EOL as last character sequence) but as a contributor I assume the project policy is as it is already, so it was left.

Reply:
Absolutely. Just pointed out because I saw it there. This PR is good to merge as-is.